src/components/Toggle/Toggle.scss

```diff
@@ -211,9 +212,10 @@
   border: 2px solid $ms-color-neutralTertiary;
   border-radius: 20px;
   cursor: pointer;
-  transition-property: background, left;
+  transition-property: background, left, border-color;
   transition-duration: 250ms;
   transition-timing-function: cubic-bezier(.4, 0, .23, 1);
+  outline: 0;
```

Comment:
The outline was going around the label area of the toggle which would not be the appropriate high contrast style. We might have to do another high contrast pass of all of the components after the mdl2 updates have been made.